code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -49,14 +49,15 @@
 	var/comp_light_color			//The color of that light
 
 
-
+/obj/item/device/modular_computer/New()
+	all_components = list()
```

Comment:
Why is this in `New()` and not simply defined with the `all_components` declaration? Should you not just have `var/list/all_components = list()` on line 42 and take it entirely away from `New()` and `Initialize()`?